specification/network/resource-manager/Microsoft.Network/preview/2021-02-01-preview/networkManagerSecurityAdminConfiguration.json

```diff
@@ -240,37 +123,37 @@
           }
         },
         "x-ms-examples": {
-          "Get security configurations": {
-            "$ref": "./examples/NetworkManagerSecurityConfigurationGet.json"
+          "Get security admin configurations": {
+            "$ref": "./examples/NetworkManagerSecurityAdminConfigurationGet.json"
           }
         }
       },
       "put": {
         "tags": [
-          "SecurityConfigurations"
+          "SecurityAdminConfigurations"
         ],
-        "description": "Creates or updates a network manager security Configuration.",
-        "operationId": "SecurityConfigurations_CreateOrUpdate",
+        "description": "Creates or updates a network manager security admin configuration.",
+        "operationId": "SecurityAdminConfigurations_CreateOrUpdate",
         "parameters": [
           {
-            "name": "securityConfiguration",
+            "name": "securityAdminConfiguration",
             "in": "body",
             "required": true,
             "schema": {
               "$ref": "#/definitions/SecurityConfiguration"
             },
-            "description": "The security configuration to create or update"
+            "description": "The security admin configuration to create or update"
           }
         ],
         "responses": {
           "200": {
-            "description": "Updated Configuration",
+            "description": "Updated Admin Configuration",
             "schema": {
               "$ref": "#/definitions/SecurityConfiguration"
             }
           },
           "201": {
```

Comment:
if this can be async i.e. non-terminal provisioning state returned in initial response you should add the x-ms-long-running-operation annotation

Reply:
No, it is not async, and it won't be long running